lib/at_cmd_parser/include/at_utils.h

```diff
@@ -0,0 +1,45 @@
+/**@file at_utils.h
```

Comment:
Do we need documentation for this file?

Reply:
I guess not, this is not a public API.

Reply:
OK. Should it be in the include folder then? (I have no idea ;) )